packages/block-editor/src/components/inserter/menu.native.js

```diff
@@ -154,6 +154,14 @@ export class InserterMenu extends Component {
 								keyExtractor={ ( item ) => item.name }
 								renderItem={ this.renderItem }
 								{ ...listProps }
+								contentContainerStyle={ [
+									...listProps.contentContainerStyle,
+									{
+										paddingBottom:
+											safeAreaBottomInset ||
+											styles.list.paddingBottom,
```

Comment:
Not a blocker, just quick idea: can you move `styles.list.paddingBottom` as default value to `safeAreaBottomInset` when destructuring it?